NGitLab/Models/Assignee.cs

```diff
@@ -22,5 +22,8 @@ public class Assignee
 
         [JsonPropertyName("created_at")]
         public DateTime CreatedAt;
+        
+        [JsonPropertyName("avatar_url")]
+        public string? AvatarURL;
```

Comment:
If you wish to use nullable you must add `#nullable enable` at the beginning of the file. Not adding nullability is fine too.

Also, you need to add it to the public API, there should be a quick fix in your IDE to do so easily.

Reply:
Ah, I should have known it wasn't as easy as blindly editing a file in the GitHub editor, my apologies 😄 I *think* I caught everything you mentioned; if there is anything missing, let me know!

Reply:
You mean you didn't test at all? 😱

Reply:
That's correct; I didn't think much of it, when I spotted that the field was missing when replacing GitLabApiClient with this in one of my work projects. I just wanted to raise a PR since I figured it'd just be a quick edit.

Reply:
Ok, I checked quickly and I feel like  there are other fields which are not correct:

```
 "assignee" : {
         "state" : "active",
         "id" : 1,
         "name" : "Administrator",
         "web_url" : "https://gitlab.example.com/root",
         "avatar_url" : null,
         "username" : "root"
      }
```
(see https://docs.gitlab.com/ee/api/issues.html#single-issue)

@meziantou @louis-z  Should we fix all the fields? The struct is very old and doesn't match the current documentation.

Reply:
Certain fields seem indeed to be missing from the GitLab response: `email`, `created_at`. Maybe we should plan a wider cleanup, to remove any irrelevant fields throughout the code base and - we can always dream - convert public fields to properties.

Reply:
What is tricky is that sometimes some DTO classes are used in multiple routes that doesn't always return the same fields. But it doesn't seem to be the case here.

Reply:
Some fields are only accessible for admin users. This is common for email addresses. Maybe this need more tests before deleting fields.